packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -251,6 +252,17 @@ export abstract class Action {
     return rule;
   }
 
+  /**
+   * The service role that is assumed during execution of action.
+   * This role is not mandatory, however more advanced configuration
```

Comment:
Docs read like this is an input, but it looks like an output.

Reply:
Changed.